command/agent/agent.go

```diff
@@ -419,13 +419,22 @@ func (a *Agent) loadKeyringFile(keyringFile string) error {
 // Stats is used to get various runtime information and stats
 func (a *Agent) Stats() map[string]map[string]string {
 	local := a.serf.LocalMember()
+	handlers := make(map[string]string)
+
+	// Convert event handlres from a string slice to a string map
+	for _, handler := range a.agentConf.EventHandlers {
```

Comment:
I think `agentConf.EventScripts()` could be called here, and you could just take the Event field since we already have a parser for this.